packages/client-core/src/components/MediaIconsBox/index.tsx

```diff
@@ -187,15 +191,17 @@ export const MediaIconsBox = () => {
               icon={<Icon type={'FlipCameraAndroid'} />}
             />
           )}
-          <IconButtonWithTooltip
-            id="UserPoseTracking"
-            title={t('user:menu.poseTracking')}
-            className={styles.iconContainer + ' ' + (isMotionCaptureEnabled ? styles.on : '')}
-            onClick={() => window.open(`/capture/${location.pathname.split('/')[2]}`, '_blank')}
-            onPointerUp={() => AudioEffectPlayer.instance.play(AudioEffectPlayer.SOUNDS.ui)}
-            onPointerEnter={() => AudioEffectPlayer.instance.play(AudioEffectPlayer.SOUNDS.ui)}
-            icon={<Icon type={'Accessibility'} />}
-          />
+          {motionCaptureEnabled && (
```

Comment:
We also need to restrict motion capture url in https://github.com/EtherealEngine/etherealengine/blob/934b9880257013987898774dabfe6a727c4ee943/packages/client/src/pages/capture/index.tsx